src/Apps/Order/routes.tsx

```diff
@@ -52,6 +58,38 @@ const confirmRouteExit = (
   return LEAVE_MESSAGING
 }
 
+const shouldRedirect = props => {
```

Comment:
Oh, nice!! I like this much better. 😄

Reply:
Extracted them into their own file

Reply:
Cool, this is a little clearer -- with vscode click to definition the distance isn't too great, and also makes things easier to unit test in isolation